gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -124,55 +112,189 @@ object Versions {
+    const val annotations = "com.google.auto.service:auto-service-annotations:${version}"
+    const val processor   = "com.google.auto.service:auto-service:${version}"
+}
+
+object AppEngine {
+    private const val version = "1.9.82"
+    private const val gradlePluginVersion = "2.2.0"
+
+    const val sdk          = "com.google.appengine:appengine-api-1.0-sdk:${version}"
+    const val gradlePlugin = "com.google.cloud.tools:appengine-gradle-plugin:${gradlePluginVersion}"
+}
+
+// https://checkerframework.org/
+object CheckerFramework {
+    private const val version = "3.7.1"
+    const val annotations = "org.checkerframework:checker-qual:${version}"
+    val dataflow = listOf(
+        "org.checkerframework:dataflow:${version}",
+        "org.checkerframework:javacutil:${version}"
     )
-    val errorProneCheckApi     = "com.google.errorprone:error_prone_check_api:${Versions.errorProne}"
-    val errorProneCore         = "com.google.errorprone:error_prone_core:${Versions.errorProne}"
-    val errorProneTestHelpers  = "com.google.errorprone:error_prone_test_helpers:${Versions.errorProne}"
-    val checkerAnnotations     = "org.checkerframework:checker-qual:${Versions.checkerFramework}"
-    val checkerDataflow        = listOf(
-        "org.checkerframework:dataflow:${Versions.checkerFramework}",
-        "org.checkerframework:javacutil:${Versions.checkerFramework}"
+}
+
+// https://errorprone.info/
+object ErrorProne {
+    private const val version = "2.5.1"
+    const val gradlePluginVersion = "1.3.0"
+    // Taken from here: https://github.com/tbroyer/gradle-errorprone-plugin/blob/v0.8/build.gradle.kts
+    const val javacPluginVersion = "9+181-r4173-1"
+
+    val annotations = listOf(
+        "com.google.errorprone:error_prone_annotations:${version}",
+        "com.google.errorprone:error_prone_type_annotations:${version}"
     )
-    val autoCommon             = "com.google.auto:auto-common:${Versions.autoCommon}"
-    val autoService            = AutoService
-    val jsr305Annotations      = "com.google.code.findbugs:jsr305:${Versions.findBugs}"
-    val guava                  = "com.google.guava:guava:${Versions.guava}"
-    val flogger                = "com.google.flogger:flogger:${Versions.flogger}"
-    val protobuf               = listOf(
-        "com.google.protobuf:protobuf-java:${Versions.protobuf}",
-        "com.google.protobuf:protobuf-java-util:${Versions.protobuf}"
+    const val core = "com.google.errorprone:error_prone_core:${version}"
+    const val checkApi = "com.google.errorprone:error_prone_check_api:${version}"
+    const val testHelpers = "com.google.errorprone:error_prone_test_helpers:${version}"
+
+    const val javacPlugin  = "com.google.errorprone:javac:${javacPluginVersion}"
+    const val gradlePlugin = "net.ltgt.gradle:gradle-errorprone-plugin:${gradlePluginVersion}"
+}
+
+// https://github.com/google/flogger
+object Flogger {
+    internal const val version = "0.5.1"
```

Comment:
Here as well, let's just keep it public.

Reply:
As discussed vocally, let's have more control of visibility so that it's easier to find usage scenarios.